wait-for-services.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+host=$1
+ports=$2
```

Comment:
This is different than the normal wait for services = ports=$* - not shift 2.

Please just use the standard: https://github.com/microservices-patterns/ftgo-application/blob/master/_wait-for-services.sh